dashboard/app/views/reports/pd_progress.html.haml

```diff
@@ -10,5 +10,11 @@
   var headers = #{ headers.try(:to_json)};
   var ht = new Handsontable(document.getElementById('progress'), {
       data: data,
-      colHeaders: headers
+      colHeaders: headers,
+      cells: function(row, col, prop) {
+        if([12, 19, 26, 34, 37, 45, 52, 59, 63, 73, 77, 81, 82].indexOf(col) != -1) {
```

Comment:
The IDs in the if-conditional are column numbers. If those aren't the IDs you are referencing, I'm not sure what you mean.

In terms of interacting with or identifying if a level is a PD Progress page, I don't know.